conduct/conduct_enforcement.md

```diff
@@ -0,0 +1,192 @@
+project.  To assist in this, the default channel of discussion for this
+committee will be a private mailing list accessible to current and future
+members of the committee as well as members of the Steering Council upon
+justified request. Only in exceptional circumstances should the committee
+deliberate outside this list (e.g. if a committee member is the target of a
+report).  In such cases, it should still provide a suitable summary of the
+discussion to the list.
+
+The Code of Conduct Committee should aim to have a resolution agreed upon within
+one week. In the event that a resolution can't be determined in that time, the
+committee will respond to the reporter(s) with an update and projected timeline
+for resolution.
+
+
+## Incident Response and Committee Actions
+
+If the act is ongoing, or involves a threat to anyone's safety (e.g. threats of
+violence), any committee member may act immediately (before reaching consensus)
+to address the situation. In ongoing situations, any member may decide to employ
+any of the tools available to the committee, including bans and blocks.
+
+If the incident involves physical danger, any member of the committee may -- and
+should -- act unilaterally to protect the safety of those involved. This can
+include contacting law enforcement (or other local personnel) and speaking on
+behalf of the Jupyter Steering Council.
+
+In situations where an individual committee member acts unilaterally, they must
+report their actions to the committee for review within 24 hours.
+
+At events where no Committee members may be present, an event organizer will act as their delegate.  They will forward any reports to the Committee, as well as a summary of any action them may have taken during the event.
+
+
+## Resolutions
+
+The committee must agree on a resolution by consensus. If the group cannot reach
+consensus and deadlocks for over a week, the group will turn the matter over to
+the Steering Council for resolution.
+
+
+Possible responses may include:
+
+* Taking no further action
+  - if we determine no violations have occurred.
+  - if the matter has been resolved publicly while the committee was
+    considering responses.
+* Coordinating voluntary mediation: if all involved parties agree, the
+  Committee may facilitate a mediation process as detailed above.
+* Remind publicly, and point out that some behavior/actions/language have been
+  judge inappropriate and why in the current context, or can but hurtful to
+  some people, requesting the community to self-adjust.
+* A private reprimand from the committee to the individual(s) involved. In this
+  case, the group chair will deliver that reprimand to the individual(s) over
+  email, cc'ing the group.
+* A public reprimand. In this case, the committee chair will deliver that
+  reprimand in the same venue that the violation occurred (i.e. in the gitter
+  chat room for a gitter chat room violation; email for an email violation,
```

Comment:
gitter a week later seems a bit silly, the relevant context in on no one's screen.  Makes more sense in mailing lists as you can attach it to the proper thread.